components/servo/collectors.go

```diff
@@ -43,6 +44,7 @@ func newPositionCollector(resource interface{}, params data.CollectorParams) (da
 		}
 		return pb.GetPositionResponse{
 			PositionDeg: pos,
+			WebhookUrl:  datacaptureWebhook.Compare(pos, pos),
```

Comment:
This can and should be changed to a UUID that maps 1 <> 1 with a set of "emission info" - emails, phones, urls, etc. This way we don't bloat our sensor data with unnecessary bytes. 

It also lets us perhaps "charge" more for data that uses our webhooks feature (because of more data required to transmit it).

Further, this embedded URL isn't the be-all-end-all of our logic - we are still able to control wether we should actually be firing the webhook in the app's datasync server logic.

Reply:
(So people with microRDK don't have "free" access to conditional webhooks)